sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_api_versions.py

```diff
@@ -10,7 +10,7 @@ class FormRecognizerApiVersion(str, Enum):
     """Form Recognizer API versions supported by this package"""
 
     #: This is the default version
-    V2_1_PREVIEW = "2.1-preview.2"
+    V2_1_PREVIEW = "2.1-preview.3"
```

Comment:
You should update the version in the readme.md

Reply:
In this PR! https://github.com/Azure/azure-sdk-for-python/pull/17406/files